ui/src/App.tsx

```diff
@@ -1,53 +1,72 @@
+import { useTheme } from '@mui/material/styles';
+import axios from 'axios';
 import { useEffect, useState } from 'react';
-import ApiService, { ServicesNames } from './api';
-import { HealthCheckApiInterface } from './api/services/healthcheck-api';
 import './App.css';
+import config from './config';
 
-const initStatusState = {
-  status: 'down',
-  db: 'down',
-};
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 
-export const App = () => {
-  const [apiStatus, setApiStatus] = useState(initStatusState);
+import AuthLayout from './components/layouts/AuthLayout.tsx';
+import MainLayout from './components/layouts/MainLayout.tsx';
+import AuthRoute from './components/routes/AuthRoute.tsx';
+import MainRoute from './components/routes/MainRoute.tsx';
+import NotFoundRoute from './components/routes/NotFoundRoute.tsx';
```

Comment:
you can add index.js file to components and export everything from there and imports will take one line, like "import {MainLayout, AuthLayout} from './components' "

Reply:
done